pages/index.vue

```diff
@@ -1,3 +1,76 @@
 <template>
-    <h1>Hello world!</h1>
-</template>
\ No newline at end of file
+    <div class="my-team">
+        <header class="my-team__main-title">
+            <h1>The Creative crew</h1>
+        </header>
+        <div class="my-team__description">
+            <h2>Who we are</h2>
+            <p>
+                We are a team of creatively diverse. driven. innovative individuals
+                working in various locations from the world
+            </p>
+        </div>
+        <div class="my-team__members">
+            <article class="my-team__members__member-card" v-for="member of members">
```

Comment:
Did you get a warning saying that you need a `key` for the `v-for`?

Reply:
No I don't have it my local

Reply:
@tracnguyen-ms you need to setup proper eslint on the repo.. as of now, this repo barely has anything apart from Nuxt.